src/app/explore/banner/banner.component.html

```diff
@@ -0,0 +1,18 @@
+<div
```

Comment:
adapted from `biggive-hero-image` in components library - but I thought especially since we don't have a precise very spec for this banner and will need to adjust it lots of times as we build it's going to be a lot more efficient to at least initially build it directly inside the angular app instead of in the components library.